config/default/jenkins-infra.yaml

```diff
@@ -294,19 +294,23 @@ jenkins:
                         value: "infra.ci.jenkins.io"
                       type: T3Medium
                       useEphemeralDevices: true
+        # To add a job, choose if you want to add it as a map, inside the corresponding arrays below
+        # - A GH Org. requires a name, the GH Org (or user handle) of the repo and the pattern to capture the repo (s) name
+        # - A Multibranch requires a name, the GH Org (or user handle) of the repo, the repo name, an string identifier (we use the YYYYMMDDXX format with XX being a suffix integer)
         jobs-settings: |
           jobs:
+            # List of GH Org. Folders, add your own as another map (or append existing repoPattern)
             - script: >
                 [
-                  ['Docker Builds', 'jenkins-infra', 'docker-.*|^jenkins.io$'],
-                  ['Terraform Jobs', 'jenkins-infra', 'aws|datadog$'],
+                  [ name: 'Docker Builds', GHOrganization: 'jenkins-infra', repoPattern: 'docker-.*|^jenkins.io$', jenkinsfilePath: 'Jenkinsfile_k8s'],
```

Comment:
Without being a blocker as it's self-explanatory but I would have love reusing Github wording with `GHOrganization` replaced by `owner` and repoPattern by `repository` :)

Reply:
But no need to spend more time on this, that's already awesome as it improves readability